.github/workflows/pull_request.yml

```diff
@@ -0,0 +1,16 @@
+jobs:
+  build-chain:
+    runs-on: ubuntu-latest
+    name: Build Pull Request
+    steps:
+    - uses: actions/checkout@v2
+    - name: Build Chain
+      id: build-chain
+      uses: Ginxo/github-action-build-chain@master
```

Comment:
Could we move this to kiegroup organization?

Reply:
yes we can, moving it

Reply:
@mareknovotny done!